backend/utils/redis/cacher.js

```diff
@@ -2,30 +2,42 @@ const croner = require("croner");
 const {
   getAllProposalRecords,
   getCurrentRoundProposals,
+  getProposalByRecordId,
 } = require("../airtable/utils");
 const redis = require("./index");
 
 async function cacheProposal(recordId, fields) {
   await redis.json.set(recordId, ".", fields);
 }
 
+function _deleteReduntantFields(proposal) {
```

Comment:
you have a type here: `_deleteReduntantFields` -> `_deleteRedundantFields`

Reply:
Thankss!